netdata.spec.in

```diff
@@ -23,6 +23,11 @@ License:	GPL v3+
 Group:		Applications/System
 Source0:	http://firehol.org/download/netdata/releases/v@PACKAGE_VERSION@/%{name}-@PACKAGE_VERSION@.tar.xz
 URL:		http://my-netdata.io/
+%if 0%{?suse_version}
+BuildRequires:	automake
+BuildRequires:	systemd-rpm-macros
```

Comment:
Correct, `systemd-rpm-macros` is not needed at runtime. Fixed this in a later commit.